### PR TITLE
[MM-19422] Add SQL trace and LDAP trace output to both console and server logs

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Adds SQL commands generated to server logs as well as LDAP debug information to server logs as well.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-19422

#### Example
Now you get the following in console and server logs (mattermost.log) :
image

### CODE REVIEW

**Comment by srkgupta** [comment]
Hi @hahmadia 
The original ticket talks about adding additional information in the logs for other types of logs as well, i.e Elasticsearch, LDAP, SQL Store. However the current PR handles it only for SQL Store. Can you please let me know if there will be additional PRs for Elasticsearch/LDAP also (or) was this missed? If latter, can you please change the logs for Elasticsearch/LDAP too.

File: store/sqlstore/supplier.go [hunk 1]
```diff
@@ -114,6 +113,16 @@ type SqlSupplier struct {
 	lockedToMaster bool
 }
 
+type TraceOnAdapter struct{}
+
+func (t *TraceOnAdapter) Printf(format string, v ...interface{}) {
+	originalString := fmt.Sprintf(format, v...)
+	newString := strings.ReplaceAll(originalString, "\n", " ")
+	newString = strings.ReplaceAll(newString, "\t", "")
```
Comment (mkraft) [inline store/sqlstore/supplier.go]: I think there's a potential to convert valid SQL to invalid SQL. See https://play.golang.org/p/c-2-8xPeqgP as a demonstration.
```suggestion
	newString = strings.ReplaceAll(newString, "\t", " ")
```

**Comment by mkraft** [comment]
If you think you may encounter 

```go
func (t *TraceOnAdapter) Printf(format string, v ...interface{}) {
        re := regexp.MustCompile(`\s`)
 	msg := fmt.Sprintf(format, v...)
 	msg = re.ReplaceAllLiteralString(msg, " ")
 	msg = strings.ReplaceAll(msg, "\"", "")
 	mlog.Debug(msg)
 }
```

**Comment by mkraft** [comment]
Sorry, pressed the wrong button. Didn't mean to close this PR.

**Comment by hahmadia** [comment]
@srkgupta Please ensure that this PR logs both SQL data and LDAP data to the mattermost.log (server logs) when you turn tracing on.

**Pull request merged**